tiledb/sm/cpp_api/array_schema_experimental.h

```diff
@@ -189,6 +189,31 @@ class ArraySchemaExperimental {
         ctx.ptr().get(), array_schema.ptr().get(), current_domain.ptr().get()));
   }
 
+  /**
+   * Retrieve an enumeration from the array schema, if has already been loaded
```

Comment:
Pretty sure you meant "if it has already".

Reply:
c1efcf0